tiledb/sm/rest/rest_client.cc

```diff
@@ -465,6 +465,35 @@ Status RestClient::get_array_metadata_from_rest(
       array->unsafe_metadata(), serialization_type_, returned_data);
 }
 
+Status RestClient::get_array_maximum_tile_size_from_rest(
```

Comment:
Are you planning to add get_array_maximum_tile_size_to_rest?

Reply:
was not aware of need/requirement... 
(assume if added, that would be *post_*...to_rest(), but not sure why that'd be useful/needed?)

Reply:
not sure why there would be a 'to' (write function?) for what is in essence a 'computed' read-only 'property'...?

Reply:
@KiterLuc good catch for most APIs, but this one is a property of the fragments that have already been written via Query, so it's not something we can write back. On the REST side the endpoint will have to call this new C API here, have the remote side do the computation against the array, and then return only the summary number here.